snmp.yml

```diff
@@ -3136,3 +3136,766 @@ servertech_sentry3:
       type: Integer
     - labelname: outletIndex
       type: Integer
+a10_axlb:
+  walk:
+  - 1.3.6.1.2.1.1.3 # Uptime
```

Comment:
While I find it super useful, please don't include any hand-edited comments or other tweaks in this file.  Annotations should only exist in the `generator.yml`.